resources/client_starter.py

```diff
@@ -253,15 +260,16 @@ def wait_for_file(waitee: Path, wait_time: int) -> Path:
 
     # get client args
     client_args = ""
-    for carg_value in args.client_args:
-        carg, value = carg_value.split(":", maxsplit=1)
-        client_args += f" --{carg} {value} "
-    LOGGER.info(f"Client Args: {client_args}")
-    if "--client-startup-json" in client_args:
-        raise RuntimeError(
-            "The '--client-args' arg cannot include \"--client-startup-json\". "
-            "This needs to be given to this script explicitly ('--client-startup-json')."
-        )
+    if args.client_args is not None:
+        for carg_value in args.client_args:
+            carg, value = carg_value.split(":", maxsplit=1)
+            client_args += f" --{carg} {value} "
+            LOGGER.info(f"Client Args: {client_args}")
+            if "--client-startup-json" in client_args:
+                raise RuntimeError(
+                    "The '--client-args' arg cannot include \"--client-startup-json\". "
+                    "This needs to be given to this script explicitly ('--client-startup-json')."
+                )
```

Comment:
Looks like this was indented one too many levels

Reply:
Maybe? this comes out of `black`.

Reply:
Oh, I'm not talking about style. When it logs `client_args` & checks for `"--client-startup-json"`, this should be outside the for-loop--look at the diff.

Reply:
Ah, right. Sorry, I had tunnel-vision.